src/importexport/guitarpro/internal/importgtp-gp5.cpp

```diff
@@ -920,7 +920,7 @@ bool GuitarPro5::read(IODevice* io)
         }
         if (barBits & SCORE_REPEAT_END) {                    // number of repeats
             bar.repeatFlags = bar.repeatFlags | mu::engraving::Repeat::END;
-            bar.repeats = readUInt8();
+            bar.repeats = readUInt8() + 1;
```

Comment:
`Measure with the end repeat barline must have a Play count value that is equal to, the total number of times of playback of this measure + 1 or more.`
https://musescore.org/en/handbook/4/voltas#change-playback